multi-agent/adapters/outbound/mongo/admin_config_reader.py

```
@@ -0,0 +1,51 @@
+        mongo_uri = f"mongodb://{mongodb_ip}:{mongodb_port}/"
+        client = pymongo.MongoClient(mongo_uri)
+        self._col = client[db_name][coll_name]
+
+    def is_admin(self, username: str) -> bool:
+        """Return *True* if *username* appears in the stored admin list."""
+        try:
+            doc = self._col.find_one({"key": "admin_users"})
+            if doc and doc.get("value"):
+                admin_usernames = doc["value"].get("admin_usernames", [])
+                return username.lower() in [u.lower() for u in admin_usernames]
+        except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)